### PR TITLE
styles-images and common css

### PR DESCRIPTION
**Pull request by Veekshith-Thoram**
all the images and common css code and the root colors have been moved to different files for importing them in the components

### CODE REVIEW

File: styles/utilities.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    --danger-background: #fd8181;
+    --light-blue: #b0ddf5;
+    --blue: #0381F7;
+    --danger-text: #DC2626;
+    --success-text: #059669;
+    --warning-text: #f59e0b;
+    --online-badge: green;
+    --offline-badge: red;
+    --busy-badge: gray;
+    --delete-btn-hover: red;
```
Comment (AtulPant2704) [inline styles/utilities.css:20]: Try to have these colors also in hex as all other colors are in hex, having same structure is a good practice.

**Review by AtulPant2704 — APPROVED** [review]
Rest all looks good.

File: styles/common.css [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+}
+
+.navigation .nav-brand {
+
+    font-weight: bold;
+    font-size: 2rem;
+
+
+}
```
Comment (AnkurChunekar) [inline styles/common.css:73]: Try to have consistent spacing and indentation.

File: styles/common.css [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+    color: var(--primary-color);
+
+}
+
+.side-nav-link {
+
+    margin-top: 1rem;
+
+}
```
Comment (AnkurChunekar) [inline styles/common.css:119]: add a empty last line at the end of the file

File: styles/utilities.css [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    --delete-btn: #ef4444;
+    --disabled-btn: #d4d4d4;
```
Comment (AnkurChunekar) [inline styles/utilities.css:22]: IMO you can improve on namings here can be more clear for eg. `--delete-btn-bg` `--delete-btn-hover` etc.

**Review by AnkurChunekar — APPROVED** [review]
Rest all looks good, great work!

**Pull request merged**